typescript/web/src/components/auth-manager/signin-modal/signin-modal.tsx

```diff
@@ -33,24 +33,35 @@ const errors = {
   EmailCreateAccount: "Try signing in with a different account.",
   Callback: "Try signing in with a different account.",
   OAuthAccountNotLinked:
-    "To confirm your identity, sign in with the same account you used originally.",
+    "We could not sign you in. If you have already signed in with this email address, try again using the same method you originally used. Password-less magic link for instance.",
   EmailSignin: "Check your email inbox.",
   CredentialsSignin:
     "Sign in failed. Check the details you provided are correct.",
   default: "Unable to sign in.",
 };
 
-const ErrorText = ({ error }: { error: string }) => (
-  <Text
-    mb="8"
-    textAlign={{ base: "center", lg: "start" }}
-    mt="12"
-    fontSize="xs"
-    color={useColorModeValue("red.600", "red.400")}
-  >
-    {error in errors ? errors[error as keyof typeof errors] : errors.default}
-  </Text>
-);
+const trimErrorMessage = (error: string): string => {
```

Comment:
I'd have named it `trimErrorKey` since it's not the message that you're trimming but a key which can be included in the errors dictionary.

Reply:
Right, I changed it